modin/pandas/series_utils.py

```diff
@@ -484,11 +484,11 @@ def second(self):
 
     @property
     def microsecond(self):
-        return Series(query_compiler=self._query_compiler.dt_microsecond())
+        return Series(query_compiler=self._query_compiler.dt_microsecond()) % 1000000
```

Comment:
Why do you need these changes?

Reply:
Omnisci returns **seconds * 1000000 + microseconds** instead of just **microseconds**. Same behavior for nanoseconds.